spec.bs

```diff
@@ -1505,6 +1506,39 @@ partial interface Navigator {
   </wpt>
 </div>
 
+<div algorithm>
+
+  The <dfn method for=Navigator>adAuctionComponents(|numAdComponents|)</dfn>
+  
+  1. Let |instance| be [=this=]'s [=relevant global object=]'s [=Window/browsing context=]'s
+     [=browsing context/fenced frame config instance=].
+
+  1. If |instance| is null, then throw a {{DOMException}}.
+
+  1. If [=this=]'s [=relevant settings object=]'s [=environment settings object/origin=] and
+    |instance|'s [=fenced frame config instance/mapped url=]'s [=url/origin=] are not [=same
+    origin=], then then throw a {{DOMException}}.
+
+  1. Let |maxAdComponents| be 40.
+
+  1. If |numAdComponents| is greater than |maxAdComponents|, then set |numAdComponents| to 
+    |maxAdComponents|.
+
+  1. Let |adComponentsURNs| be an empty [=list=] of [=fencedframeconfig/urn=]s.
+
+  1. [=map/For each=] |urn| → _ of |instance|'s [=fenced frame config instance/nested
```

Comment:
We don't really use `_` in specs I don't think? Just make this an ignored variable I think.

Reply:
Sounds good to me.